PHP >= 5.6 Required, Not >= 5.4

I tried installing and using cv as the readme directs, but it exits stating that PHP >= 5.6 is required. The readme states PHP 5.4 is required.

```
$ cv

Box Requirements Checker
========================

> Using PHP 5.4.16
> PHP is using the following php.ini file:
  /etc/php.ini

> Checking Box requirements:
  E....

 [ERROR] Your system is not ready to run the application.                       

Fix the following mandatory requirements:
=========================================

 * The application requires the version ">=5.6.0" or greater.

```

--- a/README.md
+++ b/README.md
@@ -8,7 +8,7 @@ Requirements
 
 A local CiviCRM installation.
 
-PHP v5.4+.
+PHP v5.6+.
 
 Support may vary depending on the host environment (CMS type, file-structure, symlinks, etc).
  * *Tested heavily*: Drupal 7 single-site, WordPress single-site, UnitTests

